components/notify_admin_cta/notify_admin_cta.tsx

```diff
@@ -72,21 +79,36 @@ function NotifyAdminCTA() {
         case NotifyStatus.Failed:
             return formatMessage({id: 'notify_admin_to_upgrade_cta.notify-admin.failed', defaultMessage: DafaultBtnText.Failed});
         default:
+            if (props.CtaText) {
+                return props.CtaText;
+            }
             return formatMessage({id: 'notify_admin_to_upgrade_cta.notify-admin.notify', defaultMessage: DafaultBtnText.NotifyAdmin});
         }
     };
 
     return (
-        <div>
-            <Span>{formatMessage({id: 'pricing_modal.wantToUpgrade', defaultMessage: 'Want to upgrade?'})}
-                <StyledBtn
-                    id='notify_admin_cta'
-                    onClick={notifyFunc}
-                >
-                    {btnText(notifyStatus)}
-                </StyledBtn>
-            </Span>
-        </div>);
+        <>
+            {props.CtaText ? (
+                <span>
+                    <StyledA
+                        id='notify_admin_cta'
+                        onClick={notifyFunc}
+                    >
+                        {btnText(notifyStatus)}
+                    </StyledA>
+                </span>
+            ) : (
+                <Span>
```

Comment:
Did it like that so that styling for NotifyCTA is not affected by enclosing <Span>
<img width="1521" alt="Screenshot 2022-07-06 at 13 32 44" src="https://user-images.githubusercontent.com/28563179/177531146-a600db4c-e18f-42c4-ac97-b0e6facd6e85.png">